index.html

```diff
@@ -1629,6 +1629,49 @@ <h4>DIDCommMessaging</h4>
 
       </section>
 
+      <section>
+        <h4>CredentialRegistry</h4>
+        <table class="simple" style="width: 100%;">
+          <thead>
+            <tr>
+              <th>Normative Definition</th>
+              <th>JSON-LD</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <a href="https://ssi.eecc.de/api/registry/swagger/#/Credentials/get_api_registry_vcs__id_">Verifiable Credential Registry</a>
+              </td>
+              <td>
+                <a href="https://ssi.eecc.de/api/registry/context/credentialregistry">Verifiable Credential Registry</a>
```

Comment:
you might want to give this context a shorter name, w3id.org or similar.

Reply:
Is it necessary that the context in the spec resolves? Because the given context is used within in our product. I would be quite a hassle to move it :/